src/main/sensors/gyro.c

```diff
@@ -139,20 +139,20 @@ static bool isGyroSensorCalibrationComplete(const gyroSensor_t *gyroSensor)
 
 bool gyroIsCalibrationComplete(void)
 {
-    switch (gyro.gyroToUse) {
-        default:
-        case GYRO_CONFIG_USE_GYRO_1: {
-            return isGyroSensorCalibrationComplete(&gyro.gyroSensor1);
-        }
-#ifdef USE_MULTI_GYRO
-        case GYRO_CONFIG_USE_GYRO_2: {
-            return isGyroSensorCalibrationComplete(&gyro.gyroSensor2);
-        }
-        case GYRO_CONFIG_USE_GYRO_BOTH: {
-            return isGyroSensorCalibrationComplete(&gyro.gyroSensor1) && isGyroSensorCalibrationComplete(&gyro.gyroSensor2);
+#if GYRO_COUNT > 1
+    if (gyro.gyroToUse == GYRO_CONFIG_USE_GYRO_ALL) {
+        for (int i = 0; i < GYRO_COUNT; i++) {
+            if (!isGyroSensorCalibrationComplete(&gyro.gyroSensor[i])) {
```

Comment:
You can merge all 3 copies
```suggestion
    for (int i = 0; i < GYRO_COUNT; i++) {
        if (gyro.gyroToUse == i || gyro.gyroToUse == GYRO_CONFIG_USE_GYRO_ALL) {
            if (!isGyroSensorCalibrationComplete(&gyro.gyroSensor[i])) {
```